pep-9999.rst

```diff
@@ -0,0 +1,1037 @@
+   interpreter.
+
+   * Can you ``pip install`` without an override? Currently
+     yes; after this PEP yes, since Conda would not want to
+     add the marker to prevent pip from running.
+
+   * Can it delete externally-installed packages? Currently
+     yes; after this PEP yes, unless Conda changes their
+     ``sysconfig`` schemes to use separate ones for
+     pip-installed and Conda-installed software.  Currently
+     those are the same scheme, but it may be a good idea
+     for Conda to change these, because if pip removes a
+     package, it doesn't inform Conda of that.
+
+7. A build of Python as part of an integrated monorepo with
+   its own dependency tooling, such as a private monorepo at
+   a software development company or an integrated package
+   manager like Nixpkgs_.  In these cases, the authors of
+   the monorepo may want to encourage use of the monorepo's
+   own mechanisms for importing Python packages, for the
+   same reasons as a software distribution would. They may
+   also have custom instructions for how to properly set up
+   a virtual environment.
+
+   .. _Nixpkgs: https://github.com/NixOS/nixpkgs
+
+   * Can you ``pip install`` without an override? Currently
+     yes (at least ``pip install --user``); after this PEP
+     no, because they would include a marker file.
+
+   * Can it delete externally-installed packages? Currently
+     possibly yes, depending on the details of how the
+     monorepo finds dependencies and whether it's writable;
+     after this PEP no, because they would configure their
+     default ``sysconfig`` schemes to point at a
+     non-writable location.
+
+8. A distribution Python, used in the specific context of
+   building a distribution package containing a Python
+   library - e.g., building a ``python-xyz`` RPM by using
+   ``pip install xyz`` inside the RPM spec file.
+
+   * Can you ``pip install`` without an override? Currently
+     yes; after this PEP, the build environment would need
+     to find some way to suppress the marker file to allow
+     ``pip install`` to work, but yes, provided it does
+     that.
+
+     See the `Recommendations for distributions`_ section
+     for more discussion on how to implement this.
+
+   * Can it delete externally-installed packages? Currently
+     yes (assuming the distribution doesn't have an
+     equivalent to the Debian patch mentioned earlier);
+     after this PEP no. This change is fine because a
+     package build process should not (and generally cannot)
+     include instructions to delete some other files on the
+     system; it can only package up its own files.
+
+9. A distribution Python used with ``PYTHONHOME`` to set up
+   an alternative Python environment (as opposed to a
+   virtual environment).
+
+   If the ``PYTHONHOME`` is copied directly from the
+   distribution Python (e.g., ``cp -a /usr/lib/python3.x
+   pyhome/lib``), with no modifications, then the behavior
+   is just like a distribution Python:
```

Comment:
What number is that scenario?

Reply:
case 2. will add this in a parenthetical